src/rebar_file_utils.erl

```diff
@@ -466,9 +466,10 @@ absolute_path(Path) ->
             {ok, Dir} = file:get_cwd(),
             filename:join([Dir, Path]);
         volumerelative ->
-            Volume = hd(filename:split(Path)),
+            [Letter, $: | _] = filename:nativename(filename:absname(Path)),
+            Volume = [Letter, $:],
             {ok, Dir} = file:get_cwd(Volume),
-            filename:join([Dir, Path])
+            Volume ++ filename:join([Dir, Path])
```

Comment:
So Erlang has no way to include the volume in the path? Or is this just so it works on both otp-23 and older otps?

Reply:
you can but filename:join drops it... so I have to add it after the fact.